Marlin/LED_Strip.cpp

```diff
@@ -0,0 +1,32 @@
+// RGB Led Strip Control
+ 
+#include "Marlin.h"
+#include "temperature.h" 
+#include "Configuration.h"
+#include "ultralcd.h"
+#include "LED_Strip.h"
+
+#if ENABLED(LED_STRIP)
+
+// Set color by RGB values
+void rgb_to_led_strip(int red, int green, int blue) {    
+  analogWrite(REDPIN, red);
+  analogWrite(GREENPIN, green);
+  analogWrite(BLUEPIN, blue);
```

Comment:
As it's possible to connect to non-PWM pins, and as there is now a general concept of color LEDs for Marlin, please implement this as part of the existing `set_led_color` function instead of creating a new function.